File: Misc/FizzBuzz/fizz_buzz_2.java
import java.util.ArrayList;

public class fizz_buzz_2 {
    public static void main(String[] args) {
        ArrayList<Pair> tests = new ArrayList<Pair>();
        tests.add(new Pair(3, "Fizz"));
        tests.add(new Pair(5, "Buzz"));

        for (int i = 1; i <= 100; i++) {
            String output = "";
            for (Pair pair : tests) {
                if (i % pair.key == 0) {
                    output += pair.value;
                }
            }
            System.out.println((output.length() > 0) ? output : i);
        }
    }
}

class Pair {
    int key;
    String value;

    public Pair(int key, String value) {
        this.key = key;
        this.value = value;
    }
}

File: Misc/FizzBuzz/fizz_buzz_2.js
let conditions = [
    [3, "Fizz"],
    [5, "Buzz"],
];

for (let i = 1; i <= 100; i++) {
    let output = "";
    for (const [num, val] of conditions) {
        if (i % num == 0) {
            output += val;
        }
    }
    console.log(output || i);
}
